### DISCUSSION TITLE
Try Codon to compile my python script, I got  error: cannot typecheck the program

### DISCUSSION THREAD

**Discussion by SarahYang2023** [exaloop/codon#252] [Q&A]
When I tried to use  Codon to compile my python script, I got  error: cannot typecheck the program，I've no idea how to solve it

**Comment by arshajii** [comment]
Hi @SarahYang2023 -- can you show the code that causes the error?

**Reply by SarahYang2023** [reply]
My script used  pysam and Bio.SeqIO package  of python to deal with bam and fastq file，when do compiling, I got this error.
